core/arxiv/submission/domain/compilation.py

```diff
@@ -93,6 +93,26 @@ def content_type(self):
         """Get the MIME type for the compilation product."""
         return self.output_format.content_type
 
+    @property
+    def is_succeeded(self) -> bool:
```

Comment:
Should this domain class throw an error if there are obvious logic problems? e.g. `is_succeeded` and `is_failed` can't be true at the same time, and `is_finished` and is_in_progress` can't be true at the same time.

Reply:
Fixed in d9c2ff3